src/FeaturesViewer.hpp

```diff
@@ -39,6 +40,8 @@ namespace qtAliceVision {
       Q_PROPERTY(bool trackContiguousFilter MEMBER _trackContiguousFilter NOTIFY trackContiguousFilterChanged)
       // Display only tracks with at least one inlier
       Q_PROPERTY(bool trackInliersFilter MEMBER _trackInliersFilter NOTIFY trackInliersFilterChanged)
+      // Display track endpoints
+      Q_PROPERTY(bool displayTrackEndpoints MEMBER _displayTrackEndpoints NOTIFY displayTrackEndpointsChanged)
```

Comment:
It would be great to add a display option to draw only the tracks that include the current frame.